src/main/java/racingcar/domain/CarFactory.java

```diff
@@ -0,0 +1,13 @@
+package racingcar.domain;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class CarFactory {
+    private final static String DELIMITER = ",";
+
+    public static List<Car> makeCars(String carNames) {
+        return Arrays.stream(carNames.split(DELIMITER))
+                .map(Car::new).toList();
```

Comment:
`Stream` 체이닝은 너무 편하죠 👍🏼 
```suggestion
.map(Car::new)
.toList()
```
로 한 줄씩 내리면 가독성에 도움이 될 것 같습니다!

Reply:
놓친 부분을 확인해 주셔서 감사합니다. ☺️